doxia-tools/src/site-docs/site.xml

```diff
@@ -37,6 +37,15 @@ under the License.
       <item name="Doxia Tools" href="../doxia-tools/index.html"/>
     </menu>
 
-    <menu ref="parent" />
```

Comment:
Can you explain why this is not necessary anymore?

Reply:
few lines above we have list of all modules, so link to parent in this case is duplicated.

Reply:
Alright, thanks!